src/commands/encounter/monsterUtils.ts

```diff
@@ -0,0 +1,15 @@
+import { Environment, Monster, Monsters } from '../../models/monster'
+
+/**
+ * Get only the monsters for a named environment
```

Comment:
probably don't need this function - not sure why I made it

Reply:
agreed it probably isn't needed